core/src/06-combat/01-chapter-six.md

```diff
@@ -407,7 +407,7 @@ Using a focus action involves spending all of your energy and attention on one t
 
 #### Disrupting Attack
 
-Make a damaging attack using the normal attack rules. If your attack exceeds your target defense by 10 or more, then any boons being sustained by your target immediately end.
+Make a damaging attack using the normal attack rules. If your damaging attack exceeds your target defense by 10 or more, then any boons being sustained by your target immediately end.
```

Comment:
I'd like to go with the following:

> Make a damaging attack using the normal attack rules. If your roll exceeds the target's defense by 10 or more, then any boons being sustained by your target immediately end.

Can you update in here and in the SRD accordingly?